app/src/main/java/live/hms/app2/ui/meeting/commons/VideoGridBaseFragment.kt

```diff
@@ -117,7 +117,10 @@ abstract class VideoGridBaseFragment : Fragment() {
     scalingType: RendererCommon.ScalingType = RendererCommon.ScalingType.SCALE_ASPECT_BALANCED
   ) {
     Log.d(TAG,"bindSurfaceView for :: ${item.peer.name}")
-    if (item.video == null || item.video?.isMute == true) return
+    if (item.peer.videoTrack == null
```

Comment:
If we're checking for mute you should also check for either degraded or enabled.

Reply:
Yes, you're right .. updated